instance/models/mixins/utilities.py

```diff
@@ -77,7 +77,7 @@ def provision_failed_email(self, reason, log=None):
             self.EmailBody.PROVISION_FAILED.format(name=self.name, instance_name=self.instance.name, reason=reason),
             self._get_exc_info(default=None),
             attachments=attachments,
-            extra_recipients=self.instance.additional_monitoring_emails,
+            extra_recipients=self.instance.provision_failed_emails,
```

Comment:
@swalladge, the docstring of this method has to be updated. 

Also, does it make sense to send the logs (which can be large and not easier to check than doing the same via the Ocim interface) in an attachment? I know this was already reviewed and merged. So this is just a question.

Reply:
Good catch!

> does it make sense to send the logs

probably not, but this was a decision made long ago and not in scope for changing in this ticket. :)